src/main/java/org/zalando/jackson/datatype/money/DefaultMonetaryAmountFormatFactory.java

```diff
@@ -10,8 +10,8 @@
 final class DefaultMonetaryAmountFormatFactory implements MonetaryAmountFormatFactory {
 
     @Override
-    public MonetaryAmountFormat create(final Locale locale) {
-        return MonetaryFormats.getAmountFormat(locale);
+    public MonetaryAmountFormat create(final Locale defaultLocale) {
```

Comment:
It is `SerializerProvider.getConfig().getLocale()`
1. what do you think about naming it `configLocale`?
2. what about documenting (javadoc) where this locale comes from?

Reply:
That would strictly couple the interface to the current implementation. I want to have some freedom to define where this locale may or may not come from. I could imagine that somebody specifies a static value even.

Reply:
>  I could imagine that somebody specifies a static value even.

But this parameter is under control of library, not client

Reply:
In a future version of the library. If I strictly specify where the parameter comes from, than that would  be a breaking change.

Reply:
Makes sense